clients/src/main/java/org/apache/kafka/clients/consumer/internals/FetchCollector.java

```diff
@@ -143,7 +143,7 @@ public Fetch<K, V> collectFetch(final FetchBuffer<K, V> fetchBuffer) {
         return fetch;
     }
 
-    private Fetch<K, V> fetchRecords(final CompletedFetch<K, V> nextInLineFetch) {
+    Fetch<K, V> fetchRecords(final CompletedFetch<K, V> nextInLineFetch) {
```

Comment:
seems like this could remain private as it used to be

Reply:
Done